src/core/dom/stream/detached.js

```diff
@@ -2,18 +2,18 @@ import {devAssert} from '#core/assert';
 
 export class DetachedDomStream {
   /**
-   * @param {!Window} win
-   * @param {function(!Document):void} onChunk
-   * @param {function(!Document):void} onEnd
+   * @param {Window} win
+   * @param {function(Document):void} onChunk
+   * @param {function(Document):void} onEnd
    */
   constructor(win, onChunk, onEnd) {
-    /** @const @private {function(!Document):void} */
+    /** @const @private {function(Document):void} */
     this.onChunk_ = onChunk;
 
-    /** @const @private {function(!Document):void} */
+    /** @const @private {function(Document):void} */
```

Comment:
fmi: do you know if const still has meaning? if not, we should probably remove it from everywhere.

Reply:
I do not. There's a ton of annotations and syntax fluff we can get rid of, but I'd punt on that cleanup for now.